Lib/test/test_uuid.py

```diff
@@ -512,9 +512,7 @@ def test_find_mac(self):
 
         self.assertEqual(mac, 0x1234567890ab)
 
-    def check_node(self, node, requires=None, *,
-                   # Additional bitmask checks to perform.
-                   local=False, multicast=False):
+    def check_node(self, node, requires=None, *, admin=True):
```

Comment:
The meaning of the `admin` flag is less obvious than the meaning of the opposite `network` flag. I expected this is something related to administrative privileges. Could you add an elaboration?

Reply:
For me, `network` wasn't really descriptive either.  Something like `check_universally_administered_bit` would be more descriptive but prohibitively long.  We could go back to `local` or switch the parity to `universal`.  I'm open to other suggestions.